libs/coin-modules/coin-tron/src/logic/getBalance.integ.test.ts

```diff
@@ -0,0 +1,21 @@
+import coinConfig from "../config";
+import { getBalance } from "./getBalance";
+
+describe("getBalance", () => {
+  beforeAll(() => {
+    coinConfig.setCoinConfig(() => ({
+      status: {
+        type: "active",
+      },
+      explorer: {
+        url: "https://tron.coin.ledger.com",
+      },
+    }));
+  });
+
+  it("returns correct value", async () => {
+    const result = await getBalance("TRqkRnAj6ceJFYAn2p1eE7aWrgBBwtdhS9");
+
+    expect(result).toBeUndefined();
```

Comment:
Didn't check account on explorer but should t it include native: 0 value if it's unknown account ?

Reply:
This test was used to help me develop. I will remove it.